src/app/metric_labels.rs

```diff
@@ -74,6 +74,17 @@ impl From<dst::Route> for RouteLabels {
     }
 }
 
+impl FmtLabels for dst::Route {
+    fn fmt_labels(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let labels = prefix_labels("rt", self.labels().as_ref().into_iter());
```

Comment:
What was your thinking behind the label name `rt_route`? Is the `rt` for route as well? `target_route` seems easier to understand to me, would that be technically correct?

Reply:
The `rt_route` label already exists on the route (`route_response_total` et al.) metrics.  This change just puts that label on the endpoint (`response_total` et al.) metrics as well.

The `rt` prefix indicates that this is a route label and `route` is the name of the route label.  The naming is not terribly obvious, but since the metrics are only user-facing for advanced users who are interacting directly with prometheus, the standards for clarity can be somewhat relaxed.

Reply:
I see, thanks for the explanation. Do we have a doc anywhere where we list the available user-facing labels, and what they mean? Happy to do this if there isn't one.